core/deployment/src/main/java/io/quarkus/deployment/configuration/RunTimeConfigurationGenerator.java

```diff
@@ -480,7 +480,7 @@ public void run() {
                             reinit.load(entry.getValue()));
                 }
                 final ResultHandle buildTimeConfigSource = reinit.newInstance(PCS_NEW, buildTimeValues,
-                        reinit.load("Build time config = Reloaded"), reinit.load(100));
+                        reinit.load("Build time config = Reloaded"), reinit.load(Integer.MAX_VALUE));
```

Comment:
How is this related?

Reply:
Oops, I had some unrelated changes locally and didn't notice them.